manual/source/Tutorial/InputOutput/BedIO.rst

```diff
@@ -27,12 +27,12 @@ Such an annotation track consists of multiple annotation records.
 Each annotation adds some meta information to a genomic interval (an interval with begin/end position on a contig/chromosome) The original specification of the format can be found in the `UCSC Genome Browser FAQ <https://genome.ucsc.edu/FAQ/FAQformat.html#format1>`_.
 
 The BED format is a TSV format and contains 12 columns.
-The first three column specify a genomic region (contig/chromsome name, begin, and end position) and the remaining columns contain additional information.
+The first three columns specify a genomic region (contig/chromsome name, begin, and end position) and the remaining columns contain additional information.
 The full format will be described below.
 
 Since genomic intervals are very useful and because there were many tools for manipulating BED files (sorting, intersecting intervals etc.), many other authors and projects created variants of the BED format.
 Usually, three or more columns have the same meaning as in BED and are followed by other, arbitrary tab-separated columns with additional annotation information.
-The "full" BED format is then called BED12, and BED3, BED4, BED5, and BED6 use the first 3-6 columns and keep the remaining information as data.
+The "full" BED format is then called BED12. BED3, BED4, BED5, and BED6 use the first 3-6 columns and keep the remaining information as data.
```

Comment:
It is intentional if you try to make sense of the sentence(s).
There are two information: The full BED is called BED12 ... while BED3 and so on are different formats. 
So in my opinion a "." helps to distinguish what the author is trying to tell you.
With a "," you might think that "The full BED format is called BED12, and BED3..." which is wrong.